docs/content/getting-started/setup-openfga.mdx

```diff
@@ -278,6 +278,25 @@ grpcurl -plaintext $FGA_API_HOST grpc.health.v1.Health/Check
 # {"status":"SERVING"}
 ```
 
+## Experimental Features
+Various releases of OpenFGA may have experimental features that can be enabled by providing the `--experimentals` flag or the `experimentals` config.
+
+```
+docker run -p 8080:8080 -p 8081:8081 openfga/openfga run --experimentals="feature1, feature2"
```

Comment:
Can you add an example of what this would look like when passed in as an environment variable? Just as a helpful view for our `docker-compose` using friends?

Reply:
Yep, added 👍